plugins/services/src/js/service-configuration/ServiceNetworkingConfigSection.js

```diff
@@ -12,6 +12,18 @@ import ServiceConfigUtil from '../utils/ServiceConfigUtil';
 import ServiceConfigBaseSectionDisplay from './ServiceConfigBaseSectionDisplay';
 import {findNestedPropertyInObject} from '../../../../../src/js/utils/Util';
 
+const getNetworkType = (networkType, appDefinition) => {
```

Comment:
Thanks for the research & feedback, @nLight @wavesoft @orlandohohmeier! That was a good discussion to read. I switched it to a regular `function`.